crates/pathfinder/src/sequencer.rs

```diff
@@ -44,6 +44,13 @@ pub trait ClientApi {
         contract_addr: ContractAddress,
     ) -> Result<bytes::Bytes, SequencerError>;
 
+    async fn class_by_hash(&self, class_hash: ClassHash) -> Result<bytes::Bytes, SequencerError>;
+
+    async fn class_hash(
```

Comment:
Don't we want to keep the naming convention that is used on the sequencer API?

Something like `class_hash_at` maybe?